src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -0,0 +1,116 @@
+package frc.robot.constants;
+
+import com.revrobotics.CANSparkBase.IdleMode;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.SwerveDriveKinematics;
+import edu.wpi.first.math.util.Units;
+
+//IDs and stuff
```

Comment:
Change this to a javadoc format. A javadoc docstring is like a multiline comment but it has an extra asterisk.

```java
/**
 * Software/hardware constants (e.g. CAN IDs, gear ratios, field measurements, etc.). For software configs @see RobotConfig 
 */
public class RobotConstants {

}
```